build.gradle

```diff
@@ -182,6 +182,11 @@ subprojects {
 
   task docsJar(dependsOn: javadocJar)
 
+  javadoc {
+    options.charSet = 'UTF-8'
+    options.encoding = 'UTF-8'
```

Comment:
Thank you for review, @ewencp
You're right, I should set `charSet` and `docEncoding` within javadoc.

What do you think about `encoding`? 
I thought it should be set `charSet` since I found garbled charactors in part of auto generated in javadoc such as menu (I set locale "ja_JP.UTF-8"), and I don't know where UTF-8 charactors used in code.

> Also, what is the right way to test this

I think this is too difficult, I have no ideas except of yours...